[Suggestion] Add Charged Creepers as a spawnable mob

4dTailor opened the pull request:
This adds #1.

# Feature
- Adds charged creepers as a spawnable mob

The charged creeper has a 30% chance of spawning in place of a normal creeper.
Shooting it will discharge it, turning it into a normal creeper.
The charged creeper is more powerful and deals more damage than a normal creeper.

4dTailor committed:
Added Charged Creeper Features

ZekeZDev commented:
are we able to have this only happen after a certain round instead of the start of the game.

4dTailor committed:
Added config values for minimum stage to spawn charged creepers and charged creeper percent.

## Breaking change
Modifies standard.json to add:
- "charged_spawn_stage": the stage to start spawning charged creepers (default: 3)
- "charged_spawn_percent": the chance for a creeper to be charged (default: 35)

Signed-off-by: 4dTailor <91608035+4dtailor@users.noreply.github.com>

4dTailor commented:
Added that feature, along with a way to configure charged creeper chances

**Breaking Change**:
Modifies standard.json to add:
- "charged_spawn_stage": the stage to start spawning charged creepers (default: 3)
- "charged_spawn_percent": the chance for a creeper to be charged (default: 35)

ZekeZDev commented:
branch changes closed, ill need a rebase on the main branch
